code/_helpers/items.dm

```diff
@@ -0,0 +1,9 @@
+//Prevents robots dropping their modules.
+/proc/dropsafety(var/atom/movable/A)
+	if (istype(A.loc, /mob/living/silicon))
```

Comment:
i think its good to have, has a purpose besides safety checks. i was sure lohikar did something with it. @Lohikar what was your use case aside from safety?